NAS2D/Mixer/Mixer.h

```diff
@@ -57,14 +58,14 @@ namespace NAS2D
 		 * \param loops Repeat count. -1 for continuous loop.
 		 * \param time Length of the fade in milliseconds. Default is 500.
 		 */
-		virtual void fadeInMusic(const Music& music, int loops = Mixer::CONTINUOUS, int time = Mixer::DEFAULT_FADE_TIME) = 0;
+		virtual void fadeInMusic(const Music& music, int loops = Mixer::CONTINUOUS, std::chrono::milliseconds time = std::chrono::milliseconds{Mixer::DEFAULT_FADE_TIME}) = 0;
```

Comment:
I would recommend redefining the type of `Mixer::DEFAULT_FADE_TIME` to also be `std::chrono::milliseconds`. That would simplify the default values here, as well as simplify the case of a user explicitly passing the default value.